packages/workbox-build/src/lib/rebase-path.js

```diff
@@ -0,0 +1,22 @@
+/*
+  Copyright 2018 Google LLC
+
+  Use of this source code is governed by an MIT-style
+  license that can be found in the LICENSE file or at
+  https://opensource.org/licenses/MIT.
+*/
+
+const upath = require('upath');
+
+module.exports = ({baseDirectory, file}) => {
+  // file starts our relative to the current directory, so make it absolute.
```

Comment:
s/file/File/

I'm not really sure what you mean by "file starts". Maybe reword as "Files are relative to the..."?

Reply:
Sorry, that should have been "out" instead of "our".

But I'll rephrase.